tests/test/util/test_config.cpp

```diff
@@ -58,6 +59,7 @@ TEST_CASE("Test overriding system config initialisation", "[util]")
     std::string redisPort = setEnvVar("REDIS_PORT", "1234");
 
     std::string noScheduler = setEnvVar("NO_SCHEDULER", "1");
+    std::string overrideCpuCount = setEnvVar("OVERRIDE_CPU_COUNT", "4");
```

Comment:
Gah, sorry about this but you're missing the resetting of this env var at the bottom, so all tests after this will have this set to 4

Reply:
Sure, no problem. I am very sloppy today apparently :sweat_smile: